packages/bits-ui/src/lib/bits/combobox/components/combobox-item.svelte

```diff
@@ -46,7 +46,7 @@
 		on:pointerleave
 	>
 		<slot {builder} {isSelected}>
-			{label ? label : value}
+			{label || value}
```

Comment:
should this be falsy or nullish? should the value be shown when `label = ""`?

Reply:
I think it makes sense to show the value when the label is an empty string or undefined.